lib/assessment.js

```diff
@@ -5,7 +5,13 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
-
+  minimum = arguments[0];
```

Comment:
This assigns to an external variable `minimum`! Worst case, it'll screw up operations in an outer scope; best-case, it'll create a **global** variable called `minimum`. When creating variables, use the `var` keyword.